src/components/Lend/LoanSearch/LoanSearchInterface.vue

```diff
@@ -285,8 +291,8 @@ export default {
 					await this.fetchFacets(this.loanSearchState)
 				]);
 
-				// Store loan data in component
-				this.loans = loans;
+				// Store loan data in component, guarding against null loan objects
```

Comment:
When does this happen?

Reply:
This can happen in lower envs and the vm...I'm on the fence as to if we should keep this in.

When the personalized sort gets out of sync with the db loans may not exist causing a null value to be returned instead of a loan object. When this happens no loans are shown on the page. The page loads but in a broken state.